src/extension/background-script/actions/permissions/__tests__/delete.test.ts

```diff
@@ -0,0 +1,90 @@
+import db from "~/extension/background-script/db";
+import type { DbAllowance, MessagePermissionDelete } from "~/types";
+
+import deletePermission from "../delete";
+
+const mockNow = 1487076708000;
+Date.now = jest.fn(() => mockNow);
+
+const stackerNews = {
+  host: "stacker.news",
+  imageURL: "https://stacker.news/apple-touch-icon.png",
+  name: "Stacker News",
+};
+
+const mockAllowances: DbAllowance[] = [
+  {
+    ...stackerNews,
+    enabled: true,
+    id: 3,
+    lastPaymentAt: 1667291232423,
+    lnurlAuth: true,
+    remainingBudget: 500,
+    totalBudget: 500,
+    createdAt: "1667291216372",
+    tag: "",
+    permissions: ["the-request-method-1", "the-request-method-2"],
```

Comment:
just in case you are wondering what is a real method:
https://lightning.readthedocs.io/ or https://api.lightning.community/